PKPD/gui/mainWindow.py

```diff
@@ -256,4 +285,11 @@ def _instantiate_inverse_problem(self):
 
     # show window
     window.show()
+
+    # # show our animation
```

Comment:
Could you remove these commented-out bits please?

Reply:
Thank you for noticing, it is now done.